packages/hardhat-ethers/test/fixture-projects/hardhat-project-websocket/artifacts/contracts/AmbiguousLibrary.sol/AmbiguousLibrary.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I guess this `artifacts/` folder should be added to a `.gitignore`

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1792/commits/165a656d9eb7832d8ce4048a77205e24eb83965b and  https://github.com/nomiclabs/hardhat/pull/1792/commits/4f922c9e01aa54d67e4aded0b7828b3f58be72ec